src/components/organisms/totalBalance/index.tsx

```diff
@@ -1,10 +1,22 @@
 import { Grid } from "@mui/material";
 import React from "react";
 import Typo from "../../atoms/Typo/index";
-import { customStyles } from "../../../theme/index";
 import ImageWithTypo from "../../molecules/ImageWithTypo/index";
 import explore from "../../../assets/explore.png";
+import { makeStyles } from "@material-ui/core";
 
+var customStyles = makeStyles({
+  paymentMethod: {
+    width: "250px",
+    height: "22px",
+    fontFamily: "Graphik",
+    fontStyle: "normal",
+    fontWeight: 500,
+    fontSize: "16px",
+    lineHeight: "22px",
```

Comment:
instead of hard coding font size why don't you add typography variants in theme file and use them